index.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <link rel="icon" type="image/x-icon" href="/images/favicon.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet" />
+    <title>Nova Theme</title>
+  </head>
+  <body>
+    <header class="container">
+      <nav class="container__nav">
+        <a href="/" class="logo">
+          <img src="./images/logo.svg" alt="nova Logo" loading="lazy" title="Visit Homepage" />
+        </a>
+        <ul class="nav">
+          <li class="nav__item"><a class="nav__link" href="#theteam">The Team</a></li>
+          <li class="nav__item">
+            <a class="nav__link" href="#plan">plan</a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#features">Features</a></li>
+          <li class="nav__item btn">
+            <a class="nav__link nav__link--hover" href="https://themewagon.github.io/Nova/#">
+              Download FREE!
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <section class="container__nav hero">
+        <div class="left-section__hero">
+          <h1 class="title">
+            Beautiful Free Nova
+            <br />
+            template
+          </h1>
+          <p class="description">
+            A top notch premium quality
+            <br />
+            template for your next web
+            <br />
+            project.
+          </p>
+          <div class="nav__item btn btn__hero">
+            <a class="nav__link nav__link--hover" href="https://themewagon.github.io/Nova/#">
+              Download FREE!
+            </a>
+          </div>
+        </div>
+        <div class="ipad-screen__hero">
+          <img src="./images/screen.png" alt="Next Gen App Banner" />
+        </div>
+      </section>
+    </header>
+
+    <main class="main__container">
+      <section id="features" class="features__container">
+        <h2 class="features__header">FEATURES</h2>
+        <div class="features__offer">
+          <div class="offer__item">
```

Comment:
Może `ul > li` bo to lista elementów, nawet nazwy klas o tym świadczą.

PS. W BEM jeśli występuje Element tj. `offer__item`, to musi występować Block tj. `offer` - nawet jeśli nie jest on stylowany